### ISSUE TITLE
Error: Final loader ((webpack)-rollup-loader/index.js) didn't return a Buffer or String

### ISSUE THREAD

**Issue by noahbrenner** [erikdesjardins/webpack-rollup-loader#22]
### The error I'm seeing

I'm getting the following error when building a basic "hello-world" webpack app:

```
$ npx webpack --config webpack.demo.js
Hash: 4495a4f1df16d8f60177
Version: webpack 4.28.4
Time: 692ms
Built at: 01/14/2019 8:33:42 PM
 1 asset
Entrypoint scripts = scripts.js
[0] ./src/scripts.js 896 bytes {0} [built] [failed] [1 error]
[1] ./src/lib.js 144 bytes [built]

ERROR in ./src/scripts.js
Module build failed: Error: Final loader ((webpack)-rollup-loader/index.js) didn't return a Buffer or String
    at runLoaders (D:\Harper\dev\webpack\node_modules\webpack\lib\NormalModule.js:318:18)
    at D:\Harper\dev\webpack\node_modules\loader-runner\lib\LoaderRunner.js:370:3
    at iterateNormalLoaders (D:\Harper\dev\webpack\node_modules\loader-runner\lib\LoaderRunner.js:211:10)
    at iterateNormalLoaders (D:\Harper\dev\webpack\node_modules\loader-runner\lib\LoaderRunner.js:218:10)
    at D:\Harper\dev\webpack\node_modules\loader-runner\lib\LoaderRunner.js:233:3
    at context.callback (D:\Harper\dev\webpack\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
    at D:\Harper\dev\webpack\node_modules\webpack-rollup-loader\index.js:83:3
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

### Environment

* OS: Windows 7 (32-bit)
* Terminal: Git-bash
* Node version: v10.6.0
* Package versions: see `package.json` below.

### Observations

* Running `rollup` directly against `src/scripts.js` successfully compiles the scripts.
* Running `webpack` without `webpack-rollup-loader` successfully compiles the scripts (though with the usual boilerplate).
* The most relevant info from the stack trace seems to be that [index.js at line 83][line 83] isn't returning a Buffer or String. Instead, it calls a callback (which I assume should serve the same purpose).

### Hypothesis

Since I don't see any other issues mentioning this, I wonder if this error is due to some recent `webpack` API change or if I'm just missing something obvious.

### Related files

<details>
<summary>package.json</summary>

```json
{
  "description": "A very basic project for testing webpack behavior",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "license": "MIT",
  "devDependencies": {
    "html-webpack-plugin": "^3.2.0",
    "rollup": "^1.1.0",
    "webpack": "^4.28.4",
    "webpack-cli": "^3.2.1",
    "webpack-dev-server": "^3.1.14",
    "webpack-rollup-loader": "^0.7.1"
  }
}
```

</details>

<details>
<summary>webpack.demo.js</summary>

```javascript
const path = require('path');

module.exports = {
    mode: 'production',

    entry: {
        scripts: './src/scripts.js'
    },

    output: {
        path: path.resolve(__dirname, 'dist'),
        filename: '[name].js'
    },

    module: {
        rules: [
            {
                test: /scripts\.js$/,
                use: [{
                    loader: 'webpack-rollup-loader'
                }]
            }
        ]
    }
};
```

</details>

<details>
<summary>src/scripts.js</summary>

```javascript
import {one} from './lib';

console.log('from main');
one();
```

</details>

</details>

<details>
<summary>src/lib.js</summary>

```javascript
export function one() {
    console.log('from lib.js');
}

export function two() {
    console.log('from lib.js but should not be included');
}
```

</details>

[line 83]: https://github.com/erikdesjardins/webpack-rollup-loader/blob/master/index.js#L83

**Comment by erikdesjardins** [comment]
This is a fantastic bug report!

It looks like the issue was an API change in rollup 1.0, and my rollup peerdep version range was too permissive.

Should be fixed in 0.8.0 (#23).

**Comment by noahbrenner** [comment]
Excellent, 0.8.0 is working just fine!
